test/bug.cpp~

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
This backup `bug.cpp~` file should not be included.

We may be missing `*~` or similar from `.gitignore`?